src/kernel_abi.h

```diff
@@ -1422,6 +1422,43 @@ struct BaseArch : public wordsize,
     int32_t miimon;
   } ifbond;
   RR_VERIFY_TYPE(ifbond);
+
+  typedef struct timex {
+    int modes;
+    long offset;
```

Comment:
I believe instead of `long` this should be `signed_long` so that for Arch::x86 these fields are only 4 bytes instead of 8 bytes. Right?